employees/views.py

```diff
@@ -637,7 +637,7 @@ def render_to_response(self, context: dict, **response_kwargs: Any) -> HttpRespo
 
 
 @method_decorator(login_required, name="dispatch")
-class LoadTaskActivitiesInProjectView(TemplateView):
+class LoadTaskActivities(TemplateView):
```

Comment:
We should follow to name views with `View` name at the end for easier identification.

Reply:
So please just add `View` at the end of this view name :)

Reply:
changed here and in urls